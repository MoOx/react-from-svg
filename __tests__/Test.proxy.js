// we are using this file as a proxy,
// if we don't do that, snapshots file ends up in
// lib/js/__tests__/__snapshots__
// which gets deleted on bsb -clean-world
require("../lib/js/__tests__/Test.bs.js");
