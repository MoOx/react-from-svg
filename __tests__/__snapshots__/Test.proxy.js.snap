// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clean & minimal svg all options 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg viewBox=\\"0 0 30 30\\" width={width} height={height} fill={fill} stroke={stroke}><Path d=\\"m15 3c-6.627 0-12 5.373-12 12 0 5.623 3.872 10.328 9.092 11.63-.056-.162-.092-.35-.092-.583v-2.051c-.487 0-1.303 0-1.508 0-.821 0-1.551-.353-1.905-1.009-.393-.729-.461-1.844-1.435-2.526-.289-.227-.069-.486.264-.451.615.174 1.125.596 1.605 1.222.478.627.703.769 1.596.769.433 0 1.081-.025 1.691-.121.328-.833.895-1.6 1.588-1.962-3.996-.411-5.903-2.399-5.903-5.098 0-1.162.495-2.286 1.336-3.233-.276-.94-.623-2.857.106-3.587 1.798 0 2.885 1.166 3.146 1.481.896-.307 1.88-.481 2.914-.481 1.036 0 2.024.174 2.922.483.258-.313 1.346-1.483 3.148-1.483.732.731.381 2.656.102 3.594.836.945 1.328 2.066 1.328 3.226 0 2.697-1.904 4.684-5.894 5.097 1.098.573 1.899 2.183 1.899 3.396v2.734c0 .104-.023.179-.035.268 4.676-1.639 8.035-6.079 8.035-11.315 0-6.627-5.373-12-12-12z\\"/></Svg>
  );
};
"
`;

exports[`clean & minimal svg remove-fill 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg viewBox=\\"0 0 30 30\\" width={width} height={height} fill={fill} stroke={stroke}><Path d=\\"m15 3c-6.627 0-12 5.373-12 12 0 5.623 3.872 10.328 9.092 11.63-.056-.162-.092-.35-.092-.583v-2.051c-.487 0-1.303 0-1.508 0-.821 0-1.551-.353-1.905-1.009-.393-.729-.461-1.844-1.435-2.526-.289-.227-.069-.486.264-.451.615.174 1.125.596 1.605 1.222.478.627.703.769 1.596.769.433 0 1.081-.025 1.691-.121.328-.833.895-1.6 1.588-1.962-3.996-.411-5.903-2.399-5.903-5.098 0-1.162.495-2.286 1.336-3.233-.276-.94-.623-2.857.106-3.587 1.798 0 2.885 1.166 3.146 1.481.896-.307 1.88-.481 2.914-.481 1.036 0 2.024.174 2.922.483.258-.313 1.346-1.483 3.148-1.483.732.731.381 2.656.102 3.594.836.945 1.328 2.066 1.328 3.226 0 2.697-1.904 4.684-5.894 5.097 1.098.573 1.899 2.183 1.899 3.396v2.734c0 .104-.023.179-.035.268 4.676-1.639 8.035-6.079 8.035-11.315 0-6.627-5.373-12-12-12z\\"/></Svg>
  );
};
"
`;

exports[`clean & minimal svg remove-stroke 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg viewBox=\\"0 0 30 30\\" width={width} height={height} fill={fill} stroke={stroke}><Path d=\\"m15 3c-6.627 0-12 5.373-12 12 0 5.623 3.872 10.328 9.092 11.63-.056-.162-.092-.35-.092-.583v-2.051c-.487 0-1.303 0-1.508 0-.821 0-1.551-.353-1.905-1.009-.393-.729-.461-1.844-1.435-2.526-.289-.227-.069-.486.264-.451.615.174 1.125.596 1.605 1.222.478.627.703.769 1.596.769.433 0 1.081-.025 1.691-.121.328-.833.895-1.6 1.588-1.962-3.996-.411-5.903-2.399-5.903-5.098 0-1.162.495-2.286 1.336-3.233-.276-.94-.623-2.857.106-3.587 1.798 0 2.885 1.166 3.146 1.481.896-.307 1.88-.481 2.914-.481 1.036 0 2.024.174 2.922.483.258-.313 1.346-1.483 3.148-1.483.732.731.381 2.656.102 3.594.836.945 1.328 2.066 1.328 3.226 0 2.697-1.904 4.684-5.894 5.097 1.098.573 1.899 2.183 1.899 3.396v2.734c0 .104-.023.179-.035.268 4.676-1.639 8.035-6.079 8.035-11.315 0-6.627-5.373-12-12-12z\\"/></Svg>
  );
};
"
`;

exports[`simple svg all options 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg  viewBox=\\"0 0 30 30\\"  width={width} height={height} fill={fill} stroke={stroke}><G fillRule=\\"evenodd\\" transform=\\"translate(1 2)\\"><Path d=\\"m.05081102 12.7255866c0-3.27982435 6.25117683-5.93864732 13.96239428-5.93864732 7.7112174 0 13.9623942 2.65882297 13.9623942 5.93864732 0 3.2798244-6.2511768 5.9386473-13.9623942 5.9386473-7.71121745 0-13.96239428-2.6588229-13.96239428-5.9386473zm1 0c0 2.7275396 5.80346158 4.9386473 12.96239428 4.9386473 7.1589326 0 12.9623942-2.2111077 12.9623942-4.9386473s-5.8034616-4.93864732-12.9623942-4.93864732c-7.1589327 0-12.96239428 2.21110772-12.96239428 4.93864732z\\"/><Path d=\\"m7.03185455.63388717c2.84039038-1.63994826 8.26863895 2.44424924 12.12433245 9.12231044 3.8556935 6.67806119 4.6787594 13.42114019 1.838369 15.06108839-2.8403904 1.6399483-8.2686389-2.4442492-12.12433247-9.1223104-3.85569353-6.6780612-4.67875936-13.42114018-1.83836898-15.06108843zm.500011.86601905c-2.36210126 1.36379981-1.37516805 7.49528629 2.20437703 13.69505838 3.57954512 6.1997721 8.39620112 10.1201022 10.75830242 8.7563024 2.3621012-1.3637998 1.375168-7.4952863-2.2043771-13.6950584-3.579545-6.19977207-8.39620109-10.12010219-10.75830235-8.75630238z\\"/><Path d=\\"m7.03200814 24.8173747c-2.84041121-1.6399122-2.01743103-8.3830016 1.83817767-15.06111176 3.85560869-6.67811017 9.28380539-10.76237663 12.12421659-9.12246445s2.017431 8.3830016-1.8381777 15.06111181c-3.8556087 6.6781101-9.28380535 10.7623766-12.12421656 9.1224644zm.5-.8660254c2.36211859 1.3637698 7.17872486-2.5566215 10.75819116-8.756439 3.5794663-6.19981758 4.5663217-12.3313166 2.2042031-13.69508641-2.3621186-1.3637698-7.1787249 2.5566215-10.75819118 8.75643901-3.57946633 6.1998176-4.56632167 12.3313166-2.20420308 13.6950864z\\"/><Circle cx=\\"14.105263\\" cy=\\"12.826794\\" r=\\"2.5\\"/></G></Svg>
  );
};
"
`;

exports[`simple svg remove-fill 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg  viewBox=\\"0 0 30 30\\"  width={width} height={height} fill={fill} stroke={stroke}><G fillRule=\\"evenodd\\" transform=\\"translate(1 2)\\"><Path d=\\"m.05081102 12.7255866c0-3.27982435 6.25117683-5.93864732 13.96239428-5.93864732 7.7112174 0 13.9623942 2.65882297 13.9623942 5.93864732 0 3.2798244-6.2511768 5.9386473-13.9623942 5.9386473-7.71121745 0-13.96239428-2.6588229-13.96239428-5.9386473zm1 0c0 2.7275396 5.80346158 4.9386473 12.96239428 4.9386473 7.1589326 0 12.9623942-2.2111077 12.9623942-4.9386473s-5.8034616-4.93864732-12.9623942-4.93864732c-7.1589327 0-12.96239428 2.21110772-12.96239428 4.93864732z\\"/><Path d=\\"m7.03185455.63388717c2.84039038-1.63994826 8.26863895 2.44424924 12.12433245 9.12231044 3.8556935 6.67806119 4.6787594 13.42114019 1.838369 15.06108839-2.8403904 1.6399483-8.2686389-2.4442492-12.12433247-9.1223104-3.85569353-6.6780612-4.67875936-13.42114018-1.83836898-15.06108843zm.500011.86601905c-2.36210126 1.36379981-1.37516805 7.49528629 2.20437703 13.69505838 3.57954512 6.1997721 8.39620112 10.1201022 10.75830242 8.7563024 2.3621012-1.3637998 1.375168-7.4952863-2.2043771-13.6950584-3.579545-6.19977207-8.39620109-10.12010219-10.75830235-8.75630238z\\"/><Path d=\\"m7.03200814 24.8173747c-2.84041121-1.6399122-2.01743103-8.3830016 1.83817767-15.06111176 3.85560869-6.67811017 9.28380539-10.76237663 12.12421659-9.12246445s2.017431 8.3830016-1.8381777 15.06111181c-3.8556087 6.6781101-9.28380535 10.7623766-12.12421656 9.1224644zm.5-.8660254c2.36211859 1.3637698 7.17872486-2.5566215 10.75819116-8.756439 3.5794663-6.19981758 4.5663217-12.3313166 2.2042031-13.69508641-2.3621186-1.3637698-7.1787249 2.5566215-10.75819118 8.75643901-3.57946633 6.1998176-4.56632167 12.3313166-2.20420308 13.6950864z\\"/><Circle cx=\\"14.105263\\" cy=\\"12.826794\\" r=\\"2.5\\"/></G></Svg>
  );
};
"
`;

exports[`simple svg remove-stroke 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg  viewBox=\\"0 0 30 30\\"  width={width} height={height} fill={fill} stroke={stroke}><G fillRule=\\"evenodd\\" transform=\\"translate(1 2)\\"><Path d=\\"m.05081102 12.7255866c0-3.27982435 6.25117683-5.93864732 13.96239428-5.93864732 7.7112174 0 13.9623942 2.65882297 13.9623942 5.93864732 0 3.2798244-6.2511768 5.9386473-13.9623942 5.9386473-7.71121745 0-13.96239428-2.6588229-13.96239428-5.9386473zm1 0c0 2.7275396 5.80346158 4.9386473 12.96239428 4.9386473 7.1589326 0 12.9623942-2.2111077 12.9623942-4.9386473s-5.8034616-4.93864732-12.9623942-4.93864732c-7.1589327 0-12.96239428 2.21110772-12.96239428 4.93864732z\\"/><Path d=\\"m7.03185455.63388717c2.84039038-1.63994826 8.26863895 2.44424924 12.12433245 9.12231044 3.8556935 6.67806119 4.6787594 13.42114019 1.838369 15.06108839-2.8403904 1.6399483-8.2686389-2.4442492-12.12433247-9.1223104-3.85569353-6.6780612-4.67875936-13.42114018-1.83836898-15.06108843zm.500011.86601905c-2.36210126 1.36379981-1.37516805 7.49528629 2.20437703 13.69505838 3.57954512 6.1997721 8.39620112 10.1201022 10.75830242 8.7563024 2.3621012-1.3637998 1.375168-7.4952863-2.2043771-13.6950584-3.579545-6.19977207-8.39620109-10.12010219-10.75830235-8.75630238z\\"/><Path d=\\"m7.03200814 24.8173747c-2.84041121-1.6399122-2.01743103-8.3830016 1.83817767-15.06111176 3.85560869-6.67811017 9.28380539-10.76237663 12.12421659-9.12246445s2.017431 8.3830016-1.8381777 15.06111181c-3.8556087 6.6781101-9.28380535 10.7623766-12.12421656 9.1224644zm.5-.8660254c2.36211859 1.3637698 7.17872486-2.5566215 10.75819116-8.756439 3.5794663-6.19981758 4.5663217-12.3313166 2.2042031-13.69508641-2.3621186-1.3637698-7.1787249 2.5566215-10.75819118 8.75643901-3.57946633 6.1998176-4.56632167 12.3313166-2.20420308 13.6950864z\\"/><Circle cx=\\"14.105263\\" cy=\\"12.826794\\" r=\\"2.5\\"/></G></Svg>
  );
};
"
`;

exports[`sketch export all options 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (

<Svg   viewBox=\\"0 0 97 171\\" width={width} height={height} fill={fill} stroke={stroke}><G id=\\"Page-1\\" strokeWidth=\\"1\\" fillRule=\\"evenodd\\"><Path d=\\"M9.7890625,170.355469 C12.328125,170.355469 14.4765625,169.378906 16.1367188,167.71875 L93.578125,92.0351562 C95.4335938,90.1796875 96.4101562,87.9335938 96.4101562,85.4921875 C96.4101562,82.953125 95.4335938,80.609375 93.578125,78.9492188 L16.234375,3.36328125 C14.4765625,1.60546875 12.328125,0.62890625 9.7890625,0.62890625 C4.7109375,0.62890625 0.90234375,4.53515625 0.90234375,9.61328125 C0.90234375,11.9570312 1.87890625,14.3007813 3.44140625,15.9609375 L74.5351562,85.4921875 L3.44140625,155.023438 C1.87890625,156.683594 0.90234375,158.929688 0.90234375,161.371094 C0.90234375,166.449219 4.7109375,170.355469 9.7890625,170.355469 Z\\" id=\\"chevron\\" fillRule=\\"nonzero\\"></Path></G></Svg>

  );
};
"
`;

exports[`sketch export remove-fill 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (

<Svg   viewBox=\\"0 0 97 171\\" width={width} height={height} fill={fill} stroke={stroke}><G id=\\"Page-1\\" stroke=\\"none\\" strokeWidth=\\"1\\" fillRule=\\"evenodd\\"><Path d=\\"M9.7890625,170.355469 C12.328125,170.355469 14.4765625,169.378906 16.1367188,167.71875 L93.578125,92.0351562 C95.4335938,90.1796875 96.4101562,87.9335938 96.4101562,85.4921875 C96.4101562,82.953125 95.4335938,80.609375 93.578125,78.9492188 L16.234375,3.36328125 C14.4765625,1.60546875 12.328125,0.62890625 9.7890625,0.62890625 C4.7109375,0.62890625 0.90234375,4.53515625 0.90234375,9.61328125 C0.90234375,11.9570312 1.87890625,14.3007813 3.44140625,15.9609375 L74.5351562,85.4921875 L3.44140625,155.023438 C1.87890625,156.683594 0.90234375,158.929688 0.90234375,161.371094 C0.90234375,166.449219 4.7109375,170.355469 9.7890625,170.355469 Z\\" id=\\"chevron\\" fillRule=\\"nonzero\\"></Path></G></Svg>

  );
};
"
`;

exports[`sketch export remove-stroke 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (

<Svg   viewBox=\\"0 0 97 171\\" width={width} height={height} fill={fill} stroke={stroke}><G id=\\"Page-1\\" strokeWidth=\\"1\\" fill=\\"none\\" fillRule=\\"evenodd\\"><Path d=\\"M9.7890625,170.355469 C12.328125,170.355469 14.4765625,169.378906 16.1367188,167.71875 L93.578125,92.0351562 C95.4335938,90.1796875 96.4101562,87.9335938 96.4101562,85.4921875 C96.4101562,82.953125 95.4335938,80.609375 93.578125,78.9492188 L16.234375,3.36328125 C14.4765625,1.60546875 12.328125,0.62890625 9.7890625,0.62890625 C4.7109375,0.62890625 0.90234375,4.53515625 0.90234375,9.61328125 C0.90234375,11.9570312 1.87890625,14.3007813 3.44140625,15.9609375 L74.5351562,85.4921875 L3.44140625,155.023438 C1.87890625,156.683594 0.90234375,158.929688 0.90234375,161.371094 C0.90234375,166.449219 4.7109375,170.355469 9.7890625,170.355469 Z\\" id=\\"chevron\\" fill=\\"#000000\\" fillRule=\\"nonzero\\"></Path></G></Svg>

  );
};
"
`;

exports[`with fill all options 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg  viewBox=\\"0 0 30 30\\"  width={width} height={height} fill={fill} stroke={stroke}><G fillRule=\\"evenodd\\"><Path d=\\"m0 0h30v30h-30z\\"/><Path d=\\"m10.2955556 11.5277778h-3.39250004l-1.67166667-3.17944447h-2.22888889v3.17944447h-2.99916667v-11.48861113h5.1625c3.04833334 0 4.78555556 1.475 4.78555556 4.03166666 0 1.73722223-.72111111 3.01555556-2.04861111 3.70388889zm-7.2930556-9.09583336v3.52361112h2.17972222c1.21277778 0 1.9175-.62277778 1.9175-1.78638889 0-1.13083334-.70472222-1.73722223-1.9175-1.73722223zm8.8663889-2.39277777h9.0630555v2.39277777h-6.0638888v2.14694445h5.4738888v2.37638889l-5.4738888.01638889v2.16333333h6.2277777v2.3927778h-9.2269444z\\"  fillRule=\\"nonzero\\" transform=\\"translate(7.5 15.833333)\\"/></G></Svg>
  );
};
"
`;

exports[`with fill remove-fill 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg  viewBox=\\"0 0 30 30\\"  width={width} height={height} fill={fill} stroke={stroke}><G fillRule=\\"evenodd\\"><Path d=\\"m0 0h30v30h-30z\\"/><Path d=\\"m10.2955556 11.5277778h-3.39250004l-1.67166667-3.17944447h-2.22888889v3.17944447h-2.99916667v-11.48861113h5.1625c3.04833334 0 4.78555556 1.475 4.78555556 4.03166666 0 1.73722223-.72111111 3.01555556-2.04861111 3.70388889zm-7.2930556-9.09583336v3.52361112h2.17972222c1.21277778 0 1.9175-.62277778 1.9175-1.78638889 0-1.13083334-.70472222-1.73722223-1.9175-1.73722223zm8.8663889-2.39277777h9.0630555v2.39277777h-6.0638888v2.14694445h5.4738888v2.37638889l-5.4738888.01638889v2.16333333h6.2277777v2.3927778h-9.2269444z\\"  fillRule=\\"nonzero\\" transform=\\"translate(7.5 15.833333)\\"/></G></Svg>
  );
};
"
`;

exports[`with fill remove-stroke 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg  viewBox=\\"0 0 30 30\\"  width={width} height={height} fill={fill} stroke={stroke}><G fill=\\"none\\" fillRule=\\"evenodd\\"><Path d=\\"m0 0h30v30h-30z\\" fill=\\"#dd4b39\\"/><Path d=\\"m10.2955556 11.5277778h-3.39250004l-1.67166667-3.17944447h-2.22888889v3.17944447h-2.99916667v-11.48861113h5.1625c3.04833334 0 4.78555556 1.475 4.78555556 4.03166666 0 1.73722223-.72111111 3.01555556-2.04861111 3.70388889zm-7.2930556-9.09583336v3.52361112h2.17972222c1.21277778 0 1.9175-.62277778 1.9175-1.78638889 0-1.13083334-.70472222-1.73722223-1.9175-1.73722223zm8.8663889-2.39277777h9.0630555v2.39277777h-6.0638888v2.14694445h5.4738888v2.37638889l-5.4738888.01638889v2.16333333h6.2277777v2.3927778h-9.2269444z\\"  fillRule=\\"nonzero\\" transform=\\"translate(7.5 15.833333)\\"/></G></Svg>
  );
};
"
`;

exports[`with stroke all options 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg   viewBox=\\"0 0 512 512\\" width={width} height={height} fill={fill} stroke={stroke}><Path d=\\"M256,160c16-63.16,76.43-95.41,208-96a15.94,15.94,0,0,1,16,16V368a16,16,0,0,1-16,16c-128,0-177.45,25.81-208,64-30.37-38-80-64-208-64-9.88,0-16-8.05-16-17.93V80A15.94,15.94,0,0,1,48,64C179.57,64.59,240,96.84,256,160Z\\" strokeLinecap=\\"round\\" strokeLinejoin=\\"round\\" strokeWidth=\\"32px\\"/><Line x1=\\"256\\" y1=\\"160\\" x2=\\"256\\" y2=\\"448\\" strokeLinecap=\\"round\\" strokeLinejoin=\\"round\\" strokeWidth=\\"32px\\"/></Svg>

  );
};
"
`;

exports[`with stroke remove-fill 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg   viewBox=\\"0 0 512 512\\" width={width} height={height} fill={fill} stroke={stroke}><Path d=\\"M256,160c16-63.16,76.43-95.41,208-96a15.94,15.94,0,0,1,16,16V368a16,16,0,0,1-16,16c-128,0-177.45,25.81-208,64-30.37-38-80-64-208-64-9.88,0-16-8.05-16-17.93V80A15.94,15.94,0,0,1,48,64C179.57,64.59,240,96.84,256,160Z\\" stroke=\\"#000\\" strokeLinecap=\\"round\\" strokeLinejoin=\\"round\\" strokeWidth=\\"32px\\"/><Line x1=\\"256\\" y1=\\"160\\" x2=\\"256\\" y2=\\"448\\" stroke=\\"#000\\" strokeLinecap=\\"round\\" strokeLinejoin=\\"round\\" strokeWidth=\\"32px\\"/></Svg>

  );
};
"
`;

exports[`with stroke remove-stroke 1`] = `
"import React from 'react';
import Svg, {
  Circle,
  Ellipse,
  G,
  Text,
  TSpan as Tspan,
  TextPath,
  Path,
  Polygon,
  Polyline,
  Line,
  Rect,
  Use,
  Image,
  Symbol,
  Defs,
  LinearGradient,
  RadialGradient,
  Stop,
  ClipPath,
  Pattern,
  Mask,
} from 'react-native-svg';

export default ({width, height, fill, stroke}) => {
  return (
<Svg   viewBox=\\"0 0 512 512\\" width={width} height={height} fill={fill} stroke={stroke}><Path d=\\"M256,160c16-63.16,76.43-95.41,208-96a15.94,15.94,0,0,1,16,16V368a16,16,0,0,1-16,16c-128,0-177.45,25.81-208,64-30.37-38-80-64-208-64-9.88,0-16-8.05-16-17.93V80A15.94,15.94,0,0,1,48,64C179.57,64.59,240,96.84,256,160Z\\" fill=\\"none\\" strokeLinecap=\\"round\\" strokeLinejoin=\\"round\\" strokeWidth=\\"32px\\"/><Line x1=\\"256\\" y1=\\"160\\" x2=\\"256\\" y2=\\"448\\" fill=\\"none\\" strokeLinecap=\\"round\\" strokeLinejoin=\\"round\\" strokeWidth=\\"32px\\"/></Svg>

  );
};
"
`;
